### PR TITLE
fixing vamps and ebi buttons

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_pet/handlers/api_proxy/artifact.py [hunk 1]
```diff
@@ -112,16 +112,15 @@ def artifact_summary_get_request(user_id, artifact_id):
...
                 '<a class="btn btn-primary btn-sm" '
-                'href="/ebi_submission/{{ppd_id}}">'
+                'href="/ebi_submission/%d">'
                 '<span class="glyphicon glyphicon-export"></span>'
-                ' Submit to EBI</a>')
+                ' Submit to EBI</a>' % artifact_id)
     if artifact.can_be_submitted_to_vamps:
         if not artifact.is_submitted_to_vamps:
             buttons.append(
-                '<a class="btn btn-primary btn-sm" href="/vamps/{{ppd_id}}">'
```
Comment (wasade) [inline qiita_pet/handlers/api_proxy/artifact.py]: would it be worth a `find . -name "*.py" -exec grep -H "{{.*}}" {} \;`?

**Comment by wasade** [comment]
:+1: following tests

**Comment by josenavas** [comment]
:+1: once the tests are fixed

File: qiita_pet/handlers/api_proxy/artifact.py (hunk 1 above)
Comment (antgonza) [inline qiita_pet/handlers/api_proxy/artifact.py]: Not sure ...

**Pull request merged**